### DISCUSSION TITLE
What PartitionMapping do I use for a non-partitioned asset to a specific set of partitions?

### DISCUSSION THREAD

**Discussion by dagsir** [dagster-io/dagster#17998] [Q&A]
Hello, I need to make a dependency between non-partitioned upstream asset and a set of partitioned downstream assets. Hower, there is nothing in docs about what type of PartitionMapping should be used in such case. Can someone provide working example? Thanks.

---

*The question was originally asked in [Dagster Slack](https://dagster.slack.com/archives/C01U954MEER/p1699951623723719?thread_ts=1699951623.723719&cid=C01U954MEER).*

**Comment by sryza** [comment]
To help me understand the question a little better, can you spell out the behavior that you're looking for in a little bit more detail? I.e. give a basic example with some contrived assets and partitions and specify the dependencies you're looking for?
